tests/test_mg_basic/results_true.dat

```diff
@@ -1,2 +1,2 @@
 k-combined:
-1.086852E+00 2.677280E-02
+1.073147E+00 1.602384E-02
```

Comment:
See the change to line 581 of scattdata_header.F90 and the bugfix for reading in the multiplicity matrices.